website/docs/docs/collaborate/environments/dbt-cloud-environments.md

```diff
@@ -0,0 +1,222 @@
+---
+title: "dbt Cloud environments"
+id: "dbt-cloud-environments"
+---
+
+
+An environment determines how dbt Cloud will execute your project in both the dbt Cloud IDE and scheduled jobs. Critically, in order to execute dbt, environments define three variables:
+
+1. The version of dbt Core that will be used to run your project
+2. The warehouse connection information (including the target database/schema settings)
+3. The version of your code to execute
+
+For users familiar with development on the CLI, each environment is roughly analogous to an entry in your `profiles.yml` file, with some additional information about your repository to ensure the proper version of code is executed. More info on dbt core environments [here](/docs/collaborate/environments/dbt-core-environments.md).
+
+## Types of environments
+
+In dbt Cloud, there are two types of environments: deployment and development. Deployment environments determine the settings used when jobs created within that environment are executed. Development environments determine the settings used in the dbt Cloud IDE for that particular dbt Cloud Project. Each dbt Cloud project can only have a single development environment but can have any number of deployment environments.
+
+|  | Development Environments | Deployment Environments |
+| --- | --- | --- |
+| Determines settings for | dbt Cloud IDE | dbt Cloud Job runs |
+| How many can I have in my project? | 1 | Any number |
+
+## Common environment settings
+
+Both development and deployment environments have a section called `General Settings`, which has some basic settings that all environments will define:
+
+| Setting | Example Value | Definition | Accepted Values |
+| --- | --- | --- | --- |
+| Name | Production  | The environment name  | Any string! |
+| Environment Type | Deployment | The type of environment | [Deployment, Development] |
+| dbt Version | 1.4 (latest) | The dbt version used  | Any dbt version in the dropdown |
+| Default to Custom Branch | ☑️ | Determines whether to use a branch other than the repository’s default  | See below |
+| Custom Branch | dev | Custom Branch name | See below |
+
+**dbt Version Notes**
+
+- dbt Cloud allows users to select any dbt release. At this time, **environments must use a dbt version greater than v1.0.0;** [lower versions are no longer supported](/docs/dbt-versions/upgrade-core-in-cloud).
+- If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
+
+**Custom Branch Behavior**
```

Comment:
sentence casing per [syle guide](https://github.com/dbt-labs/docs.getdbt.com/blob/mwong-styleguide-add/contributing/content-style-guide.md#location-of-ui-elements)

```suggestion
**Custom branch behavior**
```